README.md

```diff
@@ -100,6 +100,57 @@ You are free to implement the genome classes however you want, and using whateer
 
 When you have implemented the two (or more) classes, describe the complexity of each operation as a function of the genome size (at the time of the operation), and the size of the TE involved (and when copying, the offset you are copying). Put the description here:
 
+
+
 **FIXME: OPERATION COMPLEXITY**
 
+length of the genome = n, 
+length of the transcriptional element you are workign with = t, 
+offset size = f, 
+number of active TEs= a 
+**Python List**
+
+***__init__***
+__init__'s complexity is O(n) because it has to run through all of self.genome which creates a list that is as long as the length of the genome, and the rest of the elements run in constant time 
+
+***insert_te***
+insert_te's complexity is O(n*t) because the most computationally demanding portion of this operation is in line 175 when the transcriptional element that is size t gets inserted into the genome that is the size n, and in doing so it has to push the rest of the genome down. In the worst case scenario, the TE would get inserted into the first index of the genome so the operation would have to run through the size of the transcriptional element and then the whole genome size to re-index every nucleotide, giving it a complexity of O(n*t)
```

Comment:
It is not quite that bad. Usually, we can multiply the cost of an inner loop by how many times we run the outer loop, but you can only run the inner loop for one TE, so it ends up at $O(n+t)$.